test/check-machines-create

```diff
@@ -1605,8 +1600,8 @@ vnc_password= "{vnc_passwd}"
                 self._fakeFedoraTree()
 
             # Make sure we don't run into long DNS timeouts when trying to contact this server
-            self.machine.execute("echo >>/etc/hosts 127.0.0.42 mirror.i3d.net")
-            self.machine.execute("echo >>/etc/hosts ::1 mirror.i3d.net")
+            self.machine.write("/etc/hosts", "127.0.0.42 mirror.i3d.net", append=True)
+            self.machine.write("/etc/hosts", "::1 mirror.i3d.net", append=True)
```

Comment:
Same here, /etc/hosts isn't already auto-restored. Of course you can't call self.write_file() twice on the same file, so please join the two commands into one with appending both lines.

Reply:
You mentioned in chat that /etc/hosts is already restored indirectly, via setup_mock_server(). That wasn't obvious, sorry.